sycl/source/detail/queue_impl.cpp

```diff
@@ -143,7 +143,7 @@ event queue_impl::memcpy(const std::shared_ptr<detail::queue_impl> &Self,
   // pointer.
   XPTIScope PrepareNotify((void *)this,
                           (uint16_t)xpti::trace_point_type_t::node_create,
-                          SYCL_MEM_ALLOC_STREAM_NAME, "queue.memcpy()");
+                          SYCL_STREAM_NAME, "memory_transfer_node");
```

Comment:
See above comment - **we seem to have lost information.** Are memset/memcpy being recorded elsewhere, perhaps in metadata? I am assuming this was a cosmetic change and not related to HIP failure?

Reply:
it is not related to the HIP. Customer has reported the issue that after code_location enabling node_create signal for queue methods started to report different data and I found a few gaps here, this implementation is not aligned with design doc for node_create trace.